runtime/ops/http.rs

```diff
@@ -218,7 +218,7 @@ async fn op_http_request_next(
         } else {
           Cow::Owned(conn_resource.addr.to_string())
         };
-        let path = req.uri().path_and_query().unwrap();
+        let path = req.uri().path_and_query().map(|p| p.as_str()).unwrap_or("");
```

Comment:
Requests with the `CONNECT` method triggers this panic. I think we need to default to the `/` path instead of an empty string.